pkg/operator/operator.go

```diff
@@ -406,6 +406,20 @@ func New(
 	}
 	o.informers = append(o.informers, informer)
 
+	// console cluster-operator is an optional component and can be enabled
+	// after installation.
+	informer = cache.NewSharedIndexInformer(
+		o.client.ClusterOperatorListWatch(ctx, "console"),
```

Comment:
To be consistent with the way CMO infers the availability of the console operator, should it list/watch the "cluster" ClusterVersion resource instead?
btw CMO already watches the console CO but only for updates.

https://github.com/openshift/cluster-monitoring-operator/blob/70dd5c9a414448ace1d07090730510d5922b6b18/pkg/operator/operator.go#L374-L383